recipes/oasis-deconv/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "oasis-deconv" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/j-friedrich/OASIS/archive/refs/tags/v0.2.0.tar.gz
+  sha256: 3be3de589b0d070e0e0af4f215b2a16d41389be9206e90720827cec66af602b1
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - numpy
+    - cython
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib-base
+    - cython
```

Comment:
This probably isn't needed at run time, but would be needed to keep `pip check` happy.